### PR TITLE
add membership.checksum stat, same as ring.chksum

### PR DESCRIPTION
**Pull request by benfleis**
includes minor commonality changes in option handling

@dansimau @thanodnl

### CODE REVIEW

File: ringpop.go [hunk 1]
```diff
@@ -191,9 +191,23 @@ func (rp *Ringpop) startTimers() {
 	if rp.tickers != nil {
 		return
 	}
-	rp.tickers = make(chan *clock.Ticker, 1) // 1 == max number of tickers
+	_ = "breakpoint"
```
Comment (dansimau) [inline ringpop.go]: Oops, debugging breakpoint...

Comment (benfleis) [inline ringpop.go]: ack, removed.

**Comment by dansimau** [comment]
LGTM

**Comment by thanodnl** [comment]
LGTM

**Pull request merged**